services/payments/tests/unit/conftest.py

```diff
@@ -361,3 +363,62 @@ def mock_payments_gateway_service_or_none(
     mock_payments_routes(mock_payments_gateway_service_api_base)
     mock_payments_methods_routes(mock_payments_gateway_service_api_base)
     return mock_payments_gateway_service_api_base
+
+
+#
+# mock resource-usage-tracker API
+#
+
+
+@pytest.fixture
+def rut_service_openapi_specs(
+    osparc_simcore_services_dir: Path,
+) -> dict[str, Any]:
+    openapi_path = (
+        osparc_simcore_services_dir / "resource-usage-tracker" / "openapi.json"
+    )
+    return jsonref.loads(openapi_path.read_text())
+
+
+@pytest.fixture
+def mock_resource_usage_tracker_service_api_base(
+    app: FastAPI, rut_service_openapi_specs: dict[str, Any]
+) -> Iterator[MockRouter]:
+    settings: ApplicationSettings = app.state.settings
+    with respx.mock(
+        base_url=settings.PAYMENTS_RESOURCE_USAGE_TRACKER.base_url,
+        assert_all_called=False,
+        assert_all_mocked=True,  # IMPORTANT: KEEP always True!
+    ) as respx_mock:
+        assert "healthcheck" in get_in(
+            ["paths", "/", "get", "operationId"],
+            rut_service_openapi_specs,
+            no_default=True,
+        )  # type: ignore
+        respx_mock.get(
+            path="/",
+            name="healthcheck",
+        ).respond(status.HTTP_200_OK)
+
+        yield respx_mock
+
+
+@pytest.fixture
+def mock_rut_service_api(
```

Comment:
I would go for the full name like we do with all other services
```suggestion
def mock_resoruce_usage_tracker_service_api(
```